apps/exam/templates/exam/exam_intro.html

```diff
@@ -3,7 +3,7 @@
 {% block content %}
 
 <div class="container-fluid">
-	{% if exam_mode == 1 %}
+	{% if exam_mode == "practice" %}
```

Comment:
Consider creating a @property for `Attempt` class indicating the corresponding mode 

```
@property
def in_practice_mode:
  return self.mode == self.PRACTICE_MODE
```

You could also have used `exam.mode` here instead.